en/advanced_config/compass_power_compensation.md

```diff
@@ -0,0 +1,43 @@
+# Compass Power Compensation
+
+Compasses (magnetometers) should be mounted as far as possible from cables that carry large currents, as these induce magnetic fields that may corrupt the compass readings.
+
+This topic explains how to compensate for the induced magnetic fields in the cases where moving the compass is not realistic.
+
+> **Tip** Moving the compass away from power-carrying cables is the easiest and most effective way to fix this issue, because the strength of the magnetic fields decreases quadratically with the distance from the cable.
+
+## When is Power Compensation Applicable? {#when}
+
+Performing this power compensation is advisable only if all the following statements are true:
+1. The compass cannot be moved away from the power-carrying cables.
+1. There is a strong correlation between the compass readings and the thrust setpoint, and/or the battery current. 
+   ![Corrupted mag](../../images/corrupted_mag.png)
+   
+1. The drone cables are all fixed in place/do not move (calculated compensation parameters will be invalid if the current-carrying cables can move).
+
+## How to Compensate the Compass {#how}
+
+1. Make sure your drone runs a Firmware version supporting power compensation (current master, targeted for v.1.12.0).
+1. Perform the [standard compass calibration](../config/compass.html#compass-calibration).
+1. Set the parameter [SDLOG_MODE](../advanced_config/parameter_reference.md#SDLOG_MODE) to 2 to enable logging of data from boot. 
+1. Set the parameter [SDLOG_PROFILE](../advanced_config/parameter_reference.md#SDLOG_PROFILE) checkbox for *high rate* (bit 2) to get more data points.
+1. Secure the drone so that it cannot move, and attach the propellers (so the motors can draw the the same current as in flight).
+   This example secures the vehicle using straps.
+   
+   ![Corrupted mag](../../images/strap.png)
+1. Power the vehicle and switch into [ACRO flight mode](../flight_modes/acro_mc.md) (using this mode ensures the vehicle won't attempt to compensate for movement resulting from the straps).   
+   - Arm the vehicle and slowly raise the throttle to the maximum
+   - Sowly lower the throttle down to zero
+   - Disarm the vehicle
+   > **Note** Perform the test carefully and closely watch the vibrations.
+1. Retrieve the ulog and use the python script [mag_compensation.py](https://github.com/PX4/Firmware/blob/master/src/lib/mag_compensation/python/mag_compensation.py) to identify the compensation parameters.
```

Comment:
How do we use the script? Worth putting in a command line example showing typical use.

Reply:
Also worth showing how you extract the parameters from the log? This might be obvious when you run the tool, but even so, best not to leave people guessing.